ansible/roles/sonicv2/tasks/sonic-cavm.yml

```diff
@@ -0,0 +1,48 @@
+# SONiC
+
+# Setup Platform
+#- include: platform-cavm.yml
+
+# Install Cavium Host Interface Driver
+- name: Install driver package
+  become: true
+  apt: pkg=xp80PcieEndpoint
+       state=present
+       force=yes
+
+# Setup port SKU
+- name: Setup port configuration file
+  become: yes
+  template: src=etc/ssw/{{ sonic_hwsku }}/profile.ini.j2
```

Comment:
No, it is already merged in previous commit :) Here 49eee2d02c53b4a9192ce7d35de41f47e41142ea.